src/qwikql-component.tsx

```diff
@@ -14,16 +15,19 @@ import {
 interface QwikQLProps {
   url: string
   headers?: Record<string, string>
+  after$?: QRL<((resp: Response) => {})>
```

Comment:
One thing I struggled with was importing the right type for the response object returned by `rawRequest`. This `Response` interface works for my use case but it's not the correct type, which is why it's typed as `any` in the context. Maybe you know how to type this correctly?